resources/views/auth/admission/index.blade.php

```diff
@@ -5,36 +5,32 @@
 @endsection
 
 @section('content')
-    @include('auth.application.period')
+    @include('auth.admission.period')
     @if($workshops->count() > 1)
         <div class="card">
             <div class="card-content">
                 <div class="row center" style="margin-bottom: 0">
-                    <form id="workshop-filter" method="GET" route="{{route('applications')}}">
-                        <x-input.select id="workshop" :elements="$workshops" allow-empty :default="$workshop"
+                    <form id="workshop-filter" method="GET" action="{{route('admission.applicants.index')}}">
+                        <x-input.select id="workshop" :elements="$workshops" :default="$workshop"
                                         text="Műhely"/>
-                        @can('viewUnfinished', \App\Models\ApplicationForm::class)
-                            @foreach (\App\Models\ApplicationForm::STATUSES as $st)
-                                <label>
-                                    <input type="radio" name="status" value="{{$st}}"
-                                           @if($status == $st) checked @endif>
-                                    <span
-                                        style="padding-left: 25px; margin: 5px">@include('auth.application.status', ['status' => $st])</span>
-                                </label>
-                            @endforeach
-                        @endcan
+                        @can('viewUnfinished', \App\Models\Application::class)
+                            <label>
+                                <input type="checkbox" name="submitted" checked>
+                                <span style="padding-left: 25px; margin: 5px">Véglegesített</span>
+                            </label>
+                        @endif
```

Comment:
This does not seem to work for me (it does not update anything and is always checked after reload); can you check it out? Or am I missing something?

Reply:
Looks like I fixed this in #574 so you can just ignore